src/bundle/Resources/public/js/scripts/admin.anchor.navigation.js

```diff
@@ -0,0 +1,63 @@
+(function(global, doc, eZ) {
+    const sectionsPosition = {};
+    const scrollOffset = 400;
+    const formContainerNode = doc.querySelector('.ibexa-content-edit-content');
+    const isVerticalScrollVisible = () => {
+        const { scrollHeight, offsetHeight } = formContainerNode;
+
+        return scrollHeight > offsetHeight;
+    };
+    const showSection = (sectionId) => {
+        doc.querySelectorAll('.ibexa-anchor-navigation-menu__btn').forEach((btn) => {
+            const { anchorTargetSectionId } = btn.dataset;
+
+            btn.classList.toggle('ibexa-anchor-navigation-menu__btn--active', anchorTargetSectionId === sectionId);
+        });
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
+
+            section.classList.toggle('ibexa-anchor-navigation-sections__section--active', anchorSectionId === sectionId);
+        });
+    };
+    const navigateTo = (event) => {
+        let startSection = 0;
+        const { anchorTargetSectionId } = event.currentTarget.dataset;
+
+        doc.querySelectorAll('.ibexa-anchor-navigation-sections__section').forEach((section) => {
+            const { anchorSectionId } = section.dataset;
+
+            sectionsPosition[anchorSectionId] = startSection;
+            startSection = section.offsetHeight;
+        });
+
+        if (isVerticalScrollVisible()) {
+            formContainerNode.scrollTo({
+                top: sectionsPosition[anchorTargetSectionId],
+                behavior: 'smooth',
```

Comment:
What do you think? Am I missing something? @lucasOsti 
```suggestion
    const navigateTo = (event) => {
        const { anchorTargetSectionId } = event.currentTarget.dataset;
        const anchorTargetSection = doc.querySelector(`.ibexa-anchor-navigation-sections__section[data-anchor-section-id=${anchorTargetSectionId}]`);
        const anchorTargetSectionPreviousSibling = anchorTargetSection.previousSibling;
        const scrollTop = anchorTargetSectionPreviousSibling?.offsetHeight ?? 0;

        if (isVerticalScrollVisible()) {
            formContainerNode.scrollTo({
                top: scrollTop,
                behavior: 'smooth',
```